invenio_oaiserver/query.py

```diff
@@ -84,7 +58,7 @@ def get_affected_records(spec=None, search_pattern=None):
         queries.append(Q('match', **{'_oai.sets': spec}))
 
     if search_pattern:
-        queries.append(Query(search_pattern).query.accept(ElasticSearchDSL()))
+        Q('query_string', query=search_pattern)
```

Comment:
There is [another point](https://github.com/inveniosoftware/invenio-oaiserver/pull/117/files/0924a1db186cde07ac8d3a66c0bbcc824de34579#diff-bf5e6d9512655ebb58a551377859f4e4R37) where we use the query object.